programs/github

```diff
@@ -21,14 +21,62 @@ function progressbar(f)
 	return left .. right
 end
 
+-- get flag arguments
+local function findFlag(arg)
+  local accepted = {
+    { name = 'branch', flags = { '-b', '--branch' } },
+    { name = 'tag', flags = { '-t', '--tag' } },
+    { name = 'auth', flags = { '-a', '--auth' } }
+  }
+  local isFlag, flagName = false, nil
+  if arg:sub(1,1) == '-' or arg:sub(1,1) == '--' then
```

Comment:
`arg:sub(1,1)` is a string of length 1, which is never equal to a string of length 2. Just ditch the second condition